Analyzer/test/MyAnalysis.C

```diff
@@ -168,37 +186,21 @@ int main(int argc, char *argv[])
     
     try
     {
-        if(doBackground)
+        if(doBTagSF)
         {
-            printf("\n\n running AnalyzeBackground\n\n" ) ;
-            run<AnalyzeBackground>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
+            printf("\n\n running AnalyzeBTagSF\n\n") ;
+            run<AnalyzeBTagSF>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
         }
-        else if(doTopTagger)
+        if(doTrigAnalysis)
         {
-            printf("\n\n running AnalyzeTopTagger\n\n" ) ;
-            run<AnalyzeTopTagger>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
+            printf("\n\n running AnalyzeHadTrigger\n\n") ;
+            run<AnalyzeHadTrigger>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
         }
-        else if(doEventSelection)
+        else if(calcBTagSF)
         {
-            printf("\n\n running AnalyzeEventSelection\n\n") ;
-            run<AnalyzeEventSelection>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
+            printf("\n\n running CalculateBTagSF\n\n") ;
+            run<CalculateBTagSF>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
         }
-        else if(doEventShape)
-        {
-            printf("\n\n running AnalyzeEventShape\n\n") ;
-            run<AnalyzeEventShape>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
-        }
-        else if(do0Lep)
-        {
-            printf("\n\n running Analyze0Lep\n\n") ;
-            run<Analyze0Lep>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
-        }
-        else if(doStealthTT)
-        {
-            printf("\n\n running AnalyzeStealthTopTagger\n\n") ;
-            run<AnalyzeStealthTopTagger>(vvf,startFile,nFiles,maxEvts,isSkim,outfile);
-        }
-    
```

Comment:
Don't delete any of this. I just had an idea to make all of these look better that I will add after we merge everything from you.